source/MRMesh/MRMeshTopology.h

```diff
@@ -17,6 +17,22 @@ namespace MR
 class MeshTopology
 {
 public:
+    /// data of every half-edge
+    struct HalfEdgeRecord
```

Comment:
why do you move upward?

Reply:
Some public function returns it, so it shouldn't be private.

Reply:
Which one, I do not see?

Reply:
`edges()`